draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -116,6 +116,12 @@ Client implementations SHOULD implement both, but MUST implement at least one of
 
 This section discusses the needed changes to the RADIUS packet format ({{pktformat}}), port usage and shared secrets ({{portusage}}).
```

Comment:
If no changes to the packet format are needed, don't mentions this here.
"This section discusses the needed changes to the port usage and shared secrets ({{portusage}}).